### ISSUE TITLE
yAxisOffset does not work with stackData

### ISSUE THREAD

**Issue by DHerasymovLL** [Abhinandan-Kushwaha/react-native-gifted-charts#862]
### Description

Hi, first of - great library! 

I am trying to build stacked BarChart and it seems like `yAxisOffset` does not work well with it.
It does not seem to be doing anything actually, except updating the Y labels.

Here is a default behavior without `yAxisOffset` which is correct:
<img width="311" alt="Screenshot 2024-10-08 at 12 11 26 AM" src="https://github.com/user-attachments/assets/2e707020-ec4a-4785-9a74-4d2c1df4027a">

Here is the same value but with `yAxisOffset` set to 200:
<img width="310" alt="Screenshot 2024-10-08 at 12 11 34 AM" src="https://github.com/user-attachments/assets/d53e6c20-091e-449e-b767-0642eb6d4468">

And this one with `yAxisOffset` set to 400:
<img width="307" alt="Screenshot 2024-10-08 at 12 11 48 AM" src="https://github.com/user-attachments/assets/05f7c961-0323-415d-bbf7-9ac76595d93f">

* With every setting I reloaded the app.
* It looks like only Y labels get updated and bar does not scale itself.
* The props are:
```
stackData={data}
noOfSections={6}
stepValue={200}
yAxisOffset={400}
```

Is this an actual bug or am I using it wrong? It seems like it works well with regular data and same settings.

### Steps to reproduce

1. Use BarChart with `stackData`
2. Set `yAxisOffset` to any value
3. Bars are not being scaled according to `yAxisOffset`

### version of react-native-gifted-charts

1.4.43

### React Native version

0.73.8

### Platforms

Android, iOS

### Workflow

React Native

**Comment by Abhinandan-Kushwaha** [comment]
Thanks for reporting this issue @DHerasymovLL 
I will fix it soon and comment here

**Comment by Abhinandan-Kushwaha** [comment]
Hi @DHerasymovLL 
This has been fixed from versions `1.4.44` onwards. 🎉

Please use the latest version of the library.

**Comment by DHerasymovLL** [comment]
Thank you @Abhinandan-Kushwaha !
